hertzbeat-manager/src/main/java/org/apache/hertzbeat/manager/config/RestTemplateConfig.java

```diff
@@ -58,4 +60,16 @@ public ClientHttpRequestFactory simpleClientHttpRequestFactory() {
         );
     }
 
+    @Bean("restTemplateThreadPool")
+    public Executor restTemplateThreadPool() {
+        ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
+        executor.setCorePoolSize(10);
+        executor.setMaxPoolSize(50);
+        executor.setQueueCapacity(200);
```

Comment:
hi the http request require timeliness, suggest use the `SynchronousQueue` instead of  `        executor.setQueueCapacity(200);` , and ` executor.setCorePoolSize(2); executor.setMaxPoolSize(Integer.MAX_VALUE);`